.pipelines/templates/.builder-release-template.yaml

```diff
@@ -39,8 +39,8 @@ steps:
       pipeline: 318279
       specificBuildWithTriggering: true
       allowPartiallySucceededBuilds: true
-      buildVersionToDownload: 'latest'
-      downloadType: 'single'
+      buildVersionToDownload: 'latestFromBranch'
+      branchName: 'master'
```

Comment:
the example shows `branchName: 'refs/heads/master'`

Reply:
good catch